### DISCUSSION TITLE
Deploy related problem

### DISCUSSION THREAD

**Discussion by md-jihad-khan** [vercel/community#6672] [Account Issue]
### Question

when i want to deploy a project it give me a error : Your Team encountered an unknown problem. Please reach out to our support team for details.   this is my vercel id :   Hj11s5906uoMnfVlVEmQaWBB

**Comment by amyegan** [comment]
It looks like this was already fixed for you in https://github.com/vercel/community/discussions/6027#discussioncomment-9251574. If you're still having trouble, please let me know 🙏

**Comment by Shaju06** [comment]
I'm also facing same issue, kindly assist me in this

**Reply by amyegan** [reply]
It should work for you now as well

https://github.com/vercel/community/discussions/6027#discussioncomment-9423576
